README.md

```diff
@@ -92,5 +92,20 @@ You can use the flask development server for launching a local webserver:
 
 For real deployments, use a real application server (like uwsgi).
 
+## Releasing
+
+### New features
+Any new feature or bugfix should have a corresponding PR (and hopefully a GH Issue). PRs with multiple commits should be squashed prior to merging. One feature/bugfix = one PR = one commit if possible.
+
+### SemVer-ish
```

Comment:
Fine...no semver. I know we've had this discussion ad nauseam but do you want to just flip to something like YYYYMMDD?